openmmtools/mcmc.py

```diff
@@ -572,6 +572,9 @@ class BaseIntegratorMove(object):
     context_cache : openmmtools.cache.ContextCache, optional
         The ContextCache to use for Context creation. If None, the global cache
         openmmtools.cache.global_context_cache is used (default is None).
+    reassign_velocities : bool, optional
+        If True, the velocities will be reassigned from the Maxwell-Boltzmann
+        distribution at the beginning of the move (default is False).
```

Comment:
Out of curiosity, is there any reason why we *wouldn't* want to re-assign velocities?